### PR TITLE
parse 'yyxx or yyxxx' pattern for latest year

### LINKED ISSUE #13: parse pattern "1799 or 1800" for earliest and latest years
DLME openn has dates with this pattern:

```
1799 or 1800
17-- or 18--
15-- or 16--?
17th or 18th century?
```

DLME MET
```
'2nd millennium B.C. or ca. 8th century B.C.'
```

### PR DESCRIPTION
**Pull request by ndushay**
## Why was this change made?

Closes #13 
(Part of sul-dlss/dlme-transform/issues/282)

To parse earliest and latest year (as int) out of strings like the below, which occur in DLME openn collections:

```
1799 or 1800
17-- or 18--
15-- or 16--?
```

## Was the documentation (README, API, wiki, ...) updated?

yes, README

### CODE REVIEW

File: spec/parse_date/int_from_string_spec.rb [hunk 1]
```diff
@@ -379,7 +379,7 @@
     '8 B.C.' => -8
   }
 
-  describe '.earliest_year' do
+  context '.earliest_year' do
```
Comment (mjgiarlo) [inline spec/parse_date/int_from_string_spec.rb]: Hmm, what's the reason behind this change?

Comment (ndushay) [reply spec/parse_date/int_from_string_spec.rb]: uh ... mistake??? hahahah

Comment (mjgiarlo) [reply spec/parse_date/int_from_string_spec.rb]: @ndushay ok. haha

**Comment by ndushay** [comment]
@mjgiarlo fixed!  thanks for spotting that.
